packages/react-native/src/Authenticator/common/DefaultFormFields/DefaultRadioFormFields.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { censorAllButFirstAndLast, censorPhoneNumber } from '@aws-amplify/ui';
+
+import { Radio, RadioGroup } from '../../../primitives';
+import { DefaultRadioFormFieldsComponent } from './types';
+
+const censorContactInformation = (name: string, value: string): string => {
+  let censoredVal = value;
+  if (name === 'email') {
+    const splitEmail = value.split('@');
+    const censoredName = censorAllButFirstAndLast(splitEmail[0]);
+
+    censoredVal = `${censoredName}@${splitEmail[1]}`;
+  } else if (name === 'phone_number') {
+    censoredVal = censorPhoneNumber(value);
+  }
+  return censoredVal;
+};
+
+const DefaultFormFields: DefaultRadioFormFieldsComponent = ({
```

Comment:
This should have been name `DefaultRadioFormFields`, will address in a follow up PR